traffic-policy/actions/terminate-tls/config.mdx

```diff
@@ -0,0 +1,28 @@
+## Configuration Reference
+
+This is the [Traffic Policy](/docs/http/traffic-policy/) configuration reference for this action.
+
+### Action Type
+
+`terminate-tls`
+
+### Configuration Fields
+
+| Parameter                              | Type       | Description                                                                                                                                          |
+| -------------------------------------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `min_version`                          | `string`   | The minimum TLS version to support. Must be one of `1.0`, `1.1`, `1.2`, or `1.3`. Defaults to `1.2`.                                                 |
+| `max_version`                          | `string`   | The maximum TLS version to support. Must be one of `1.0`, `1.1`, `1.2`, or `1.3`. Defaults to `1.3`.                                                 |
+| `server_private_key`                   | `string`   | The PEM-encoded private key for the server if using a custom certificate (must be specified with `server_certificate`).                              |
+| `server_certificate`                   | `string`   | The PEM-encoded certificate for the server if using a custom certificate (must be specified with `server_private_key).                               |
+| `mutual_tls_certificate_authorities`   | `[]string` | A list of PEM-encoded certificates that are trusted as Certificate Authorities for mutual TLS authentication.                                        |
+| `mutual_tls_certificate_authority_ids` | `[]string` | A list of [Certificate Authority](/docs/api/resources/certificate-authorities/) IDs that are trusted for mutual TLS authentication.                  |
```

Comment:
so in the most recent iteration of back and forth.... we landed on a single string array where they are either IDs or PEMs
`mutual_tls_certificate_authorities`